docs/getting-started.md

```diff
@@ -109,3 +109,62 @@ You can now continue using your UP address within the dApp:
 ```javascript
 const myUPAddress = myContracts.ERC725Account.address;
 ```
+
+## Proxy Deployment
+
+LSPFactory uses proxy deployment of smart contracts to maximise gas efficiency. This can be configured inside the `options` object when deploying [Universal Profiles](./deployment/universal-profile.md) or [Digital Assets](./deployment/digital-asset.md).
+
+When using proxy deployment contracts will be deployed as a **minimal proxy contract** based on [EIP1167](https://eips.ethereum.org/EIPS/eip-1167). This is a lightweight contract that inherits its logic by referencing the address of a contract already deployed on the blockchain. Inheriting allows cheaper deployment of Universal Profiles and Digital Assets because only the proxy contract needs to be deployed.
```

Comment:
Suggestion:

_This is a lightweight contract that ~~inherits~~ ..._ -> _This is a lightweight contract that **derives** ..._

_~~Inheriting~~ allows cheaper deployment of Universal Profiles_ -> _**This** allows cheaper deployment of Universal Profiles_